examples/notebooks/knn_mnist.ipynb

```diff
@@ -104,19 +105,23 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "CPU times: user 24.2 s, sys: 681 ms, total: 24.9 s\n",
-      "Wall time: 1.1 s\n"
-     ]
+     "data": {
+      "text/plain": [
+       "'Intel(R) extension for Scikit-learn time: 2.516653537750244 s'"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "%%time\n",
+    "start = time()\n",
     "from sklearn.neighbors import KNeighborsClassifier\n",
     "knn = KNeighborsClassifier(**params).fit(x_train, y_train)\n",
-    "predicted = knn.predict(x_test)"
+    "predicted = knn.predict(x_test)\n",
+    "sklearnex_time = time() - start\n",
+    "f\"Intel(R) extension for Scikit-learn time: {sklearnex_time} s\""
```

Comment:
```suggestion
    "f\"Intel(R) extension for Scikit-learn time: {time() - start} s\""
```

Reply:
I think using variable is better in this case. We can use this for create summary